packages/z/zafiro-icons-light/package.yml

```diff
@@ -0,0 +1,19 @@
+name       : Zafiro-Icons-Light
+version    : 1.3
+release    : 1
+source     :
+    - https://github.com/zayronxio/Zafiro-icons/releases/download/1.3/Zafiro-Icons-Light.tar.xz : eda5cad186239edbb0c8d8fb57d71be3de305f8e31eaef37e8e4cc22ebef1317
+homepage   : https://github.com/zayronxio/Zafiro-icons/
+license    : GPL-3.0-or-later
+component  : desktop.theme
+summary    : The Zafiro-Icons
+description: |
+    Minimalist icons created with the flat-design technique, utilizing washed-out colors and always accompanied by white. The priority is simplicity.
+install : |
+    mv /home/build/YPKG/root/Zafiro-Icons-Light/build/Zafiro-Icons-Light/apps/scalable/βTORRENT.svg /home/build/YPKG/root/Zafiro-Icons-Light/build/Zafiro-Icons-Light/apps/scalable/TORRENT.svg
+    install -dm00644 $installdir/usr/share/icons/Zafiro-Icons-Light
+    cp -r * $installdir/usr/share/icons/Zafiro-Icons-Light/
+    chmod -R 00644 $installdir/usr/share/icons/Zafiro-Icons-Light/*
```

Comment:
Why do you need to change the permission? The folder permission is already set in previous step.